doc/arkode/guide/source/Usage/MRIStep/Custom_Inner_Stepper/Description.rst

```diff
@@ -416,5 +456,86 @@ following member functions:
       value if a recoverable error occurred, or a negative value if it failed
       unrecoverably.
 
+   **Notes:**
+      If provided, the :c:type:`MRIStepInnerResetFn` function will always be called
+      *before* the function :c:type:`MRIStepInnerEvolveFn` is first called.
+
+
    **Example codes:**
       * ``examples/arkode/CXX_parallel/ark_diffusion_reaction_p.cpp``
+
+
+.. c:type:: int (*MRIStepInnerGetAccumulatedError)(MRIStepInnerStepper stepper, sunrealtype* accum_error)
+
+   This function returns an estimate of the accumulated solution error arising from the inner stepper.
+
+   **Arguments:**
+      * *stepper* -- the inner stepper object.
```

Comment:
I couldn't get that syntax to work with `.. c:type::` blocks (as opposed to `.. c:function::` blocks above)

Reply:
I went ahead and converted this to the new formatting, but it looks messy for `.. c:type::` blocks, placing separate "parameter" sections for each input (instead of condensing them together like with `.. c:function::`.  Perhaps this is something that we can control in our Sphinx configuration, since our other "new" formatted "User supplied" functions also look poor (e.g., https://sundials.readthedocs.io/en/latest/cvode/Usage/index.html#user-supplied-functions vs https://sundials.readthedocs.io/en/latest/arkode/Usage/User_supplied.html).